collectors/python.d.plugin/hpssa/README.md

```diff
@@ -8,44 +8,66 @@ sidebar_label: "HP Smart Storage Arrays"
 
 Monitors controller, cache module, logical and physical drive state and temperature using `ssacli` tool.
 
+Executed commands:
+
+- `sudo -n ssacli ctrl all show config detail`
+
 ## Requirements:
 
 This module uses `ssacli`, which can only be executed by root. It uses
-`sudo` and assumes that it is configured such that the `netdata` user can
-execute `ssacli` as root without password.
+`sudo` and assumes that it is configured such that the `netdata` user can execute `ssacli` as root without a password.
 
-Add to `sudoers`:
+- add to the `sudoers`
 
-```
+```bash
 netdata ALL=(root)       NOPASSWD: /path/to/ssacli
 ```
 
-To collect metrics, the module executes: `sudo -n ssacli ctrl all show config detail`
+- reset netdata systemd
+  unit [CapabilityBoundingSet](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Capabilities) (Linux
+  distributions with systemd)
+
+Default CapabilityBoundingSet doesn't allow using `sudo` and is quite strict in general.
+
+> :warning: Resetting it is not an optimal solution,
+> but we couldn't find exact set of capabilities to execute ssacli with sudo.
+
+As the `root` user do the following:
+
+```cmd
+mkdir /etc/systemd/system/netdata.service.d
+echo -e '[Service]\nCapabilityBoundingSet=~' | tee /etc/systemd/system/netdata.service.d/unset-capability-bounding-set.conf
+systemctl daemon-reload
+systemctl restart netdata.service
+```
 
-This module produces:
+## Charts
 
-1.  Controller state and temperature
-2.  Cache module state and temperature
-3.  Logical drive state
-4.  Physical drive state and temperature
+- Controller status
+- Controller temperature
+- Logical drive status
+- Physical drive status
+- Physical drive temperature
 
 ## Enable the collector
 
-The `hpssa` collector is disabled by default. To enable it, use `edit-config` from the Netdata [config
-directory](/docs/configure/nodes.md), which is typically at `/etc/netdata`, to edit the `python.d.conf` file.
+The `hpssa` collector is disabled by default. To enable it, use `edit-config` from the
+Netdata [config directory](/docs/configure/nodes.md), which is typically at `/etc/netdata`, to edit the `python.d.conf`
+file.
 
 ```bash
 cd /etc/netdata   # Replace this path with your Netdata config directory, if different
 sudo ./edit-config python.d.conf
 ```
 
-Change the value of the `hpssa` setting to `yes`. Save the file and restart the Netdata Agent with `sudo systemctl
-restart netdata`, or the appropriate method for your system, to finish enabling the `hpssa` collector.
+Change the value of the `hpssa` setting to `yes`. Save the file and restart the Netdata Agent
+with `sudo systemctl restart netdata`, or the appropriate method for your system, to finish enabling the `hpssa`
+collector.
 
```

Comment:
We did not add an example, like we did for others:

```yaml
megacli: yes
```

Please, also add an example here.